### PR TITLE
VSM skeleton

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -246,6 +246,44 @@
...
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "VSM development (Chrome)",
+            "url": "http://localhost:1338",
+            "webRoot": "${workspaceFolder}",
+            "preLaunchTask": "VSM Serve (Dev)"
+        },
+        {
+            /*
+            Launch the playground to be use to test changes in dev packages
```
Comment (RaananW) [inline .vscode/launch.json:267]: nit - you can remove this comment, or change "playground" with "VSM"

File: .vscode/launch.json [hunk 2]
```diff
@@ -246,6 +246,44 @@
...
+            "type": "pwa-msedge",
+            "request": "launch",
+            "name": "Launch VSM (Edge)",
+            "url": "http://localhost:1338",
+            "webRoot": "${workspaceFolder}",
+            "preLaunchTask": "VSM Serve for core (Dev)"
+        },
+        {
+            /*
+            Launch the playground to be use to test changes in dev packages
```
Comment (RaananW) [inline .vscode/launch.json:278]: Same as above :-)

**Review by RaananW — COMMENTED** [review]
nit-picking FTW! :-)

LGTM, just a few small changes to avoid errors in the future.

File: packages/tools/vsm/public/index.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        versions = versions.map((v) => v.replace("https://preview.babylonjs.com", "https://babylonsnapshots.z22.web.core.windows.net/" + snapshot));
+    }
+
+    return new Promise((resolve) => {
+        loadInSequence(versions, 0, resolve);
+    });
+};
+
+checkBabylonVersionAsync().then(() => {
+    loadScriptAsync("babylon.guiEditor.js").then(() => {
```
[the remark this answers is not in the document]
Comment (carolhmj) [reply packages/tools/vsm/public/index.js:78]: Thanks for catching!

**Review by RaananW — COMMENTED** [review]
Just one more direct comment regarding package name.

I also think we might need to mention somewhere (maybe in the index.ts or some readme.md for the package) that this is obviously an unfinished work and just a skeleton that will be replaced when we start working on it.

File: packages/tools/vsm/package.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@tools/vsm",
```
[the remark this answers is not in the document]
Comment (carolhmj) [reply packages/tools/vsm/package.json:2]: That makes sense.

**Review by RaananW — APPROVED** [review]

**Pull request merged**